TypeScript.YAML-tmLanguage

```diff
@@ -241,6 +241,8 @@ repository:
     end: (?=$|[,);\}\]]|//)|(?==[^>])|(?<=[\}>\]\)]|[a-zA-Z_$])\s*(?=\{)
     patterns:
     - include: '#type'
+    - include: '#string'
```

Comment:
I was fixing "Colorize string at certain syntactically incorrect positions." https://github.com/Microsoft/TypeScript-Sublime-Plugin/pull/307 so I think people may want to have string here too.